Formula/oak.rb

```diff
@@ -0,0 +1,23 @@
+class Oak < Formula
+  desc "Expressive, simple, dynamic programming language"
+  homepage "https://oaklang.org/"
+  url "https://github.com/thesephist/oak/archive/v0.2.tar.gz"
+  sha256 "d06ba53f88490f5d8f70f45515afaf6ee8e4119c8ab477e3b4de8ef3cc7a2c05"
+  license "MIT"
+  head "https://github.com/thesephist/oak.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
```suggestion
```

Reply:
Fixed in a force-push to keep everything in a single commit